packages/lib/zuauth/src/index.ts

```diff
@@ -9,6 +10,7 @@ import {
 } from "./zuauth";
 
 export {
+  EDGE_ESMERALDA_TICKET,
```

Comment:
will this fit in the URL?

Reply:
It does, although I'm always a bit surprised that it works. If there wasn't already something very much like this in production, I'd be really worried about it.